### DISCUSSION TITLE
Reporting a Server

### DISCUSSION THREAD

**Discussion by kittysmugglers** [mastodon/mastodon#27915] [Show and tell]
I'd like to report a mastodon server for CSA, how would I go about doing that?

**Comment by ThisIsMissEm** [comment]
Report it to your law enforcement, and their webhost / CDN; Chances are they are already on a blocklist for distributing this type of content.

**Reply by kittysmugglers** [reply]
Thank you, I already sent a request to fbi.gov

**Reply by ThisIsMissEm** [reply]
Cool, I guess we can close this discussion thread then?
